clean-abap/CleanABAP.md

```diff
@@ -4606,6 +4606,15 @@ by redirecting them to test data defined in the unit test without interfering wi
 
 - Use the CDS test environment (`CL_CDS_TEST_ENVIRONMENT`) to test your CDS views.
 
+- Use the authorization double (`CL_AUNIT_AUTHORITY_CHECK`) to withdraw authorizations during the runtime of the test.
```

Comment:
It's not an "authorization double" though, is it? The ABAP Doc of the class calls it an "AUTHORITY-CHECK test environment", so I think we should go with that.

Reply:
Alternatively, "authority check test environment" would also work..